src/components/ChatView.vue

```diff
@@ -45,15 +68,107 @@ export default {
 		},
 	},
 
+	data: function() {
+		return {
+			isDraggingOver: false,
+		}
+	},
+
+	methods: {
+		/**
+		 * Uploads and shares the selected files
+		 * @param {object} event the file input event object
+		 */
+		async processFiles(event) {
```

Comment:
Maybe this can be deduplicated from NewMessageForm. It looks rather complex and is almost the same content?
```diff
--- ChatView.vue
+++ NewMessageForm.vue
@@ -1,14 +1,14 @@
 
 		async processFiles(event) {
-			// restore non dragover state
-			this.isDraggingOver = false
 			// Store the token in a variable to prevent changes when changing conversation
 			// when the upload is still running
 			const token = this.token
 			// Create a unique id for the upload operation
 			const uploadId = new Date().getTime()
 			// The selected files array coming from the input
-			const files = Object.values(event.dataTransfer.files)
+			const files = Object.values(event.target.files)
+			// Clear the input for the next uploads
+			this.$refs.fileUploadInput.value = ''
 			// Process these files in the store
 			await this.$store.dispatch('uploadFiles', { uploadId, token, files })
 			// Get the files that have successfully been uploaded from the store
```